src/content-helper/dashboard-widget/top-posts/component-list-item.tsx

```diff
@@ -0,0 +1,91 @@
+/**
+ * External dependencies
+ */
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { TopPostData } from './model';
+import { impreciseNumber } from '../../../blocks/shared/functions';
+import PublishedLinkIcon from '../../../blocks/content-helper/icons/published-link-icon';
+import { getSmartShortDate } from '../../../blocks/shared/utils/date';
+
+interface TopPostListItemProps {
+	post: TopPostData;
+}
+
+/**
+ * Returns a single list item depicting a post.
+ *
+ * @param {TopPostData} post The Post to be shown.
+ */
+function TopPostListItem( { post }: TopPostListItemProps ): JSX.Element {
+	return (
+		<li className="parsely-top-post">
+			<div className="parsely-top-post-wrapper">
+
+				<div className="parsely-top-post-thumbnail">
+					<span className="screen-reader-text">Thumbnail</span>
+					<img src={ post.thumbUrlMedium } alt={ __( 'Post thumbnail', 'wp-parsely' ) } />
```

Comment:
What wil happen if there is no thumbnail?

Reply:
@abelsonlive, I guess is it possible for the `analytics/posts` endpoint to return an empty thumb URL?

Reply:
@acicovic yes, it could be null for a number of reasons, most commonly because a post has only recently been published and has not yet been crawled. You might consider using the `image_url` as a fallback, though that will have different dimensions.